### ISSUE TITLE
[Accessibility][Screen reader - Property Pane_Ask a question]: Screen reader is not reading any information while navigating over the syntax suggestions available in "Prompt(?)" edit field.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2078]
**User Experience** 
If a screen reader will not provide any information about the syntax suggestions available in "Prompt(?)" edit field, then screen reader dependent users will never get to know about code creation ideas.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen and navigate t"Prompt(?)" edit field.
5 .After '-' type "if" and navigate over the syntax suggestions.
6. Observer the issue.

**Actual:** 
Screen reader is not reading any information while navigating over the syntax suggestions available in "Prompt(?)" edit field. Screen reader keeps silent.

**Expected:**
Screen reader should announce the syntax suggestions available in "Prompt(?)" edit field as "If suggestion 1 of 3"

**MAS Impacted:** [MAS1.3.1](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment:**
[MAS1.3.1_Syntax suggestions.zip](https://github.com/microsoft/BotFramework-Composer/files/4249580/MAS4.2.1_Syntax.suggestions.zip)
![MAS4 2 1_Syntax suggestions](https://user-images.githubusercontent.com/60913131/75241216-b8591b80-57eb-11ea-8192-9da4cc58cf86.png)

**Comment by beyackle** [comment]
We're using the React version of the Monaco editor as provided here: https://github.com/suren-atoyan/monaco-react. On its own demo page (https://monaco-react.surenatoyan.com/), the autocomplete suggestions just get read as the names themselves with no indication of which number the user is on or how many there are in the box.

**Comment by beyackle** [comment]
I'm marking this as Blocked until I can actually get some help from the Monaco team on this. I have no idea what's preventing our version of the editor from reading out suggestions.

**Comment by Amit8527510735** [comment]
#Nonbenchmark;
